config/webpacker.yml

```diff
@@ -76,6 +76,9 @@ test:
   <<: *default
   compile: true
 
+  # Extract and emit a css file
+  extract_css: false
```

Comment:
I was trying to run `bundle exec rspec` and a lot of tests were failing with "Webpacker::Manifest::MissingEntryError". I ran `RAILS_ENV=test rails webpacker:compile` but it said there is nothing to do. The tests pass when extract_css is set to false though.